Segmentation fault in print() function

**Describe the bug**
Invocation of `print()` with many parameters (and maybe under certain circumstances) leads to segfault .

```
error: ./tests/main.cx:99 function 'print' not found in package 'main'
unexpected fault address 0xfffffffffffffff8
fatal error: fault
[signal SIGSEGV: segmentation violation code=0x1 addr=0xfffffffffffffff8 pc=0x4382619]

goroutine 1 [running, locked to thread]:
runtime.throw(0x44d6fc6, 0x5)
	/usr/local/Cellar/go/1.11/libexec/src/runtime/panic.go:608 +0x72 fp=0xc0000db228 sp=0xc0000db1f8 pc=0x402c4f2
runtime.sigpanic()
	/usr/local/Cellar/go/1.11/libexec/src/runtime/signal_unix.go:397 +0x275 fp=0xc0000db278 sp=0xc0000db228 pc=0x4041395
main.yyParse(0x4552160, 0xc0000bc050, 0xc0000bc050)
	$GOPATH/src/github.com/skycoin/cx/cxgo/cxgo.go:2270 +0x8da9 fp=0xc0000dba68 sp=0xc0000db278 pc=0x4382619
main.main()
	$GOPATH/src/github.com/skycoin/cx/cxgo/cxgo.nn.go:8435 +0x5f7 fp=0xc0000dbf98 sp=0xc0000dba68 pc=0x438ab77
runtime.main()
	/usr/local/Cellar/go/1.11/libexec/src/runtime/proc.go:201 +0x207 fp=0xc0000dbfe0 sp=0xc0000dbf98 pc=0x402de67
runtime.goexit()
	/usr/local/Cellar/go/1.11/libexec/src/runtime/asm_amd64.s:1333 +0x1 fp=0xc0000dbfe8 sp=0xc0000dbfe0 pc=0x4058db1

goroutine 19 [runnable]:
main.NewLexerWithInit.func1(0xc000574060, 0xc000082480, 0xc000744230, 0x48327c0, 0x75, 0x75, 0x60, 0x4)
	$GOPATH/src/github.com/skycoin/cx/cxgo/cxgo.nn.go:171 +0x6c7
created by main.NewLexerWithInit
	$GOPATH/src/github.com/skycoin/cx/cxgo/cxgo.nn.go:209 +0x178
make: *** [test] Error 2
```

**To Reproduce**
Steps to reproduce the behavior:
1. Add the statement `print("Comparing %d against %d", flags, flag)` in the for loop inside function `printFlags()` of `main.cx` 
1. Run `make test`

**Expected behavior**
Not sure , but definitely not segfault . @amherag what should be expected from this ?

**Desktop (please complete the following information):**
 - OS: MacOS
 - CX Version : `CX version 0.5.18` running from `develop` branch

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -35,6 +35,7 @@
   * #104: Dubious error message when indexing an array with a substraction expression
   * #105: Dubious error message when inline initializing a slice
   * #131: Problem with struct literals in short variable declarations
+  * #199: Trying to call an undefined function no longer throws a segfault
   * #218: Type checking now works with receiving variables of unexpected types
 * Documentation
   * CONTRIBUTING.md: Information about how to contribute to CX

--- a/cxgo/cxgo.y
+++ b/cxgo/cxgo.y
@@ -1145,7 +1145,7 @@ expression_statement:
                 { $$ = nil }
 	|       expression SEMICOLON
                 {
-			if $1[len($1) - 1].Operator == nil && !$1[len($1) - 1].IsMethodCall {
+			if len($1) > 0 && $1[len($1) - 1].Operator == nil && !$1[len($1) - 1].IsMethodCall {
 				outs := $1[len($1) - 1].Outputs
 				if len(outs) > 0 {
 					println(CompilationError(outs[0].FileName, outs[0].FileLine), "invalid expression")

